src/Tizen.NUI/src/public/BaseComponents/ViewLiteProperty.cs

```diff
@@ -47,6 +47,9 @@ internal void SetBackgroundColor(L.Color color)
 
                 Object.InternalSetPropertyMap(SwigCPtr, Property.BACKGROUND, map.SwigCPtr);
             }
+
+            NotifyPropertyChanged(nameof(BackgroundColor));
+            NotifyBackgroundChanged();
```

Comment:
Do this two lines emit different two event?

Reply:
Right. Second one notify the background has been change for one of possible way : `BackgroundColor`, `BackgroundImage` and `Background()`.

Reply:
I understood it is a historical issue.